getting-started/templates/systemlink-values.yaml

```diff
@@ -987,6 +1014,133 @@ fileingestion:
     ##
     encryptionKeySecret: "fileingestionservices-download-encryption-key"
 
+## Configuration for FileIngestionCDC.
+## <ATTENTION> - Needed if FileIngestion search capabilities are enabled.
+##
+fileingestioncdc:
```

Comment:
Please exclude values we don't expect a typical user to need to change from these template files. We want to keep the cognitive load down for users doing their first deployment. If the user is unlikely to change the default and reading the comments related to the value is not important for initial setup, it should probably not be in the template.

Reply:
Excluded values that are unexpected to be configured on the first run, though this raises another issue. We don't have any documentation on our helm charts anywhere..

Reply:
We should determine where in our public ni.com docs to describe these options.